modules/ols-1-0-known-issues.adoc

```diff
@@ -0,0 +1,31 @@
+// This module is used in the following assemblies:
+
+// * lightspeed-docs-main/release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-1-0-known-issues_{context}"]
+= Known issues
+
+The following issues are identified with {ols-official} 1.0:
+
+* For {ocp-product-title} 4.17 and later, the {ols-short} icon disappears when you click *Create Namespace* or *Create Project* from one of the following locations in the {ocp-product-title} web console:
++
+--
+** *Administration -> Namespaces* 
+** *Home -> Projects -> Create Project*
+** The *Project* drop-down menu located at the top of most pages
+--
++
+Workaround: Refresh the web browser and the {ols-short} icon appears. link:https://issues.redhat.com/browse/OLS-1815[OLS-1815].
+
+* Changing the value of the `quota` parameter in the `spec.ols.quotaHandlersConfig.limitersConfig` specification of the `OLSConfig` custom resource file does not take effect until the currently defined quota period expires.
++
+Workaround: Delete the {ols-short} Operator. Ensure that any operand pods that the {ols-short} Operator manages, and the Persistent Volume Claim `lightspeed-postgres-pvc` associated with the `postgres` pod are also deleted. Then, install the {ols-short} Operator again. https://issues.redhat.com/browse/OLS-1826[OLS-1826]
+
+* After restarting the `postgres` pod, the {ols-short} service pod fails to connect to the `postgres` pod.
++
+Workaround: Restart the {ols-short} service pod so that the service pod connects to `postgres` pod. link:https://issues.redhat.com/browse/OLS-1815[OLS-1835].
+
+* When using the BYO Knowledge tool to create a retrieval-augmented generation (RAG) database, you must create `OLSConfig` custom resource file with BYO Knowledge index before you create the {ocp-product-title} index.
```

Comment:
Would just add an article here:
```suggestion
* When using the BYO Knowledge tool to create a retrieval-augmented generation (RAG) database, you must create a `OLSConfig` custom resource file with BYO Knowledge index before you create the {ocp-product-title} index.
```

Reply:
The BYO Knowledge feature was pulled from the release so I am removing this entry from the release notes.